app/imports/components/Users/AdminPanel/UsersManagement/UsersList/setAdmin/setAdmin.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import { PropTypes } from 'prop-types';
+import { actions } from './actions.js';
+
+
+export default class SetAdmin extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.handleOpen = this.handleOpen.bind(this);
+        this.handleClose = this.handleClose.bind(this);
+
+        this.users = props.users;
+        this.userId = props.userId;
+        this.index = actions.getIndexOfUserById(this.users, this.userId);
+        this.newAdmin = this.users[this.index];
```

Comment:
What's about `state`? 
And you can get user using [lodash#find](https://lodash.com/docs/4.17.4#find)